foundation/foundation-drift-detection/aissemble-foundation-drift-detection-client-python/src/drift/detection/rest/client/drift_rest_client.py

```diff
@@ -68,10 +68,10 @@ def invoke_drift(self, policy_identifier, input, control):
         query_params = {"policyIdentifier": policy_identifier}
         data_headers = {"Content-type": "application/json"}
         # uncomment below for debugging purposes
-        # print('JSON being sent to drift invocation service: ' + drift_data_input.json())
+        # print('JSON being sent to drift invocation service: ' + drift_data_input.json(serialize_as_any=True))
         return requests.post(
             hostname + "/invoke-drift",
             params=query_params,
-            data=drift_data_input.json(),
+            data=drift_data_input.model_dump_json(serialize_as_any=True),
```

Comment:
I(Informational): This `serialize_as_any` setting `true` will dump base class and any child classes. 
This was defaulted to true in V1. 
In V2 it became defaulted to false in which made json to dump only base classes.
We need this to true since we need DriftVariables and DriftVariable class that is child classes of DriftData.

Reply:
S: I'd prefer to use the [recommended method](https://docs.pydantic.dev/latest/concepts/serialization/#overriding-the-return-type-when-dumping-a-model) of specifying the actual schema if that's possible instead of relying on duck typing.  The change to the default was to make the behavior align better with production standards it seems.